packages/scanner/test/cli/commands.spec.ts

```diff
@@ -49,6 +52,7 @@ describe('commands', () => {
           let watcher: watchScan.Watcher | undefined;
           jest.spyOn(watchScan, 'default').mockImplementation((opts) => {
             watcher = new watchScan.Watcher(opts);
+            clearInterval(watcher.interval);
```

Comment:
FWIW, we shouldn't need to call `clearInterval` anywhere due to the `unref`.

Reply:
I tried [removing](https://github.com/getappmap/appmap-js/pull/847/commits/24843d05939c70e55e4c26fa19497fc76ef6ad78) the `clearInterval` and it sent telemetry messages on CI.  This makes me believe `withStubbedTelemetry` doesn't mock telemetry.

Reply:
I stepped through with a debugger and can confirm `Telemetry.sendEvent` is stubbed. Maybe it's something else?

Reply:
Ah, I see. The mocks are fine, but with the `setInterval` it's possible for telemetry to be emit after the `afterEach` hook is run to restore the stubbed methods. I'm a little worried about this leaking telemetry into other test cases.

Reply:
I'm not sure how to ensure telemetry won't leak into other testcases.

I looked into (implemented but didn't push on this branch) enabling the timer when an event is added to the batch and disabling it when a batch is sent.  I think it can still leak telemetry into other testcases if the batch contains telemetry that didn't get sent.